sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/service/SampleNavigationTypeService.kt

```diff
@@ -18,8 +18,7 @@ package br.com.zup.beagle.sample.spring.service
 
 import br.com.zup.beagle.core.CornerRadius
 import br.com.zup.beagle.core.Style
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/b2ede46dae081f3fd943db0aa8e6d6633f8e3d3f/sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/service/SampleNavigationTypeService.kt#L20"></span>The import 'br.com.zup.beagle.core.Style' is unused.</td>
    </tr>
  </tbody>
</table>